backend/src/handlers/crudHandler.ts

```diff
@@ -0,0 +1,67 @@
+import { Model, Document } from 'mongoose';
+import { Response, Request, Router } from 'express';
+
+interface Handler<T extends Document> {
+  model: Model<T>;
+  req: Request;
+  res: Response;
+}
+
+const getAllHandler = <T extends Document>({ model, res }: Handler<T>): void => {
+  model.find({}, (err, product) => {
+    if (err) res.status(500).send('Error getting all pathways');
```

Comment:
Is there a way to make the message here use the type instead of always saying 'pathways'?